test/lib/renderIndex.test.js

```diff
@@ -0,0 +1,102 @@
+const chai = require('chai')
+const sinon = require('sinon')
+const rewire = require('rewire')
+const fs = require('fs')
+const path = require('path')
+
+const cssFolder = path.join(__dirname, '..', '..', 'dist', 'static', 'css')
+const jsFolder = path.join(__dirname, '..', '..', 'dist', 'static', 'js')
+
+chai.use(require('sinon-chai'))
+chai.should()
+
+let lastTpl
+let lastOptions
+
+const mockResponse = {
+  render: (tpl, options) => {
+    lastTpl = tpl
+    lastOptions = options
+  }
+}
+
+describe('#renderIndex', () => {
+  describe('Processing configuration', () => {
+    let renderIndex
+
+    beforeEach(() => {
+      renderIndex = rewire('../../lib/renderIndex')
+      renderIndex.__set__('webConfig', {
+        base: '/configured/path'
+      })
+    })
+
+    it('uses the base from the `X-External-Path` header', () => {
+      renderIndex(
+        {
+          headers: {
+            'x-external-path': '/test/base'
+          }
+        },
+        mockResponse
+      )
+      lastOptions.config.base.should.equal('/test/base/')
```

Comment:
safer to return the bool
```suggestion
      return lastOptions.config.base.should.equal('/test/base/')
```